frontend/src/entities/dataset/detail/dataset-detail.tsx

```diff
@@ -55,7 +62,7 @@ function DatasetDetail () {
     datasetDetails.set('Access level', showAccessLevel(dataset));
     datasetDetails.set('Location', dataset.location);
     if (dataset.type === DatasetType.GROUP) {
-        datasetDetails.set(`Group${dataset.groups && dataset.groups.length > 1 ? 's' : ''}`, dataset.groups);
+        datasetDetails.set(`Group${dataset.groups && dataset.groups.length > 1 ? 's' : ''}`, dataset.groups?.length === 0 ? 'None' : dataset.groups);
```

Comment:
I'm not sure if this actually occurs but it looks like `dataset.groups?.length` could return undefined. In which case it isn't `=== 0` and would try and display `dataset.groups`. Might be better to let truthiness rule the day…

```typescript
dataset.groups?.length ? dataset.groups : 'None'
```

Reply:
good point, will do